html/modules/custom/bc_dc/bc_dc.module

```diff
@@ -276,3 +276,22 @@ function bc_dc_cron(): void {
   $reviewReminder = \Drupal::service('bc_dc.review_reminder');
   $reviewReminder->sendRemindersToAllUsers();
 }
+
+/**
+ * Implements hook_form_alter.
+ */
+function bc_dc_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  if ($form['form_id']['#id'] === 'edit-bc-dc-add-columns-form') {
+    $nid = $form_state->getStorage('nid');
+    $paragraph = \Drupal::entityTypeManager()->getStorage('node')->load($nid['nid']);
+    $paragraph_field_items = $paragraph->get('field_columns')->referencedEntities();
+    if ($paragraph_field_items) {
+      $form['data_dictionary'] = [
+        '#markup' =>
+          '<p><a href="/node/' . $nid['nid'] . '/download/columns/csv">Download current data dictionary file to CSV</a></p>
```

Comment:
These should be generated with `Link::createFromRoute()`.

Reply:
fixed